File: src/main/java/io/github/falon/scryingstatues/ScryingStatuesClient.java
package io.github.falon.scryingstatues;

import io.github.falon.scryingstatues.entity.statue.StatueRenderer;
import io.github.falon.scryingstatues.init.ModEntities;
import net.fabricmc.api.ClientModInitializer;
import net.fabricmc.fabric.api.client.rendering.v1.EntityRendererRegistry;

public class ScryingStatuesClient implements ClientModInitializer {

	@Override
	public void onInitializeClient() {
		// This code runs on the client side. However, since we don't have any client-side code yet, this method is empty.
		EntityRendererRegistry.register(ModEntities.STATUE, StatueRenderer::new);
	}
}

File: src/main/java/io/github/falon/scryingstatues/ScryingStatuesClient.kt
package io.github.falon.scryingstatues

import io.github.falon.scryingstatues.entity.statue.StatueRenderer
import io.github.falon.scryingstatues.init.ModEntities
import net.fabricmc.fabric.api.client.rendering.v1.EntityRendererRegistry
import org.quiltmc.loader.api.ModContainer
import org.quiltmc.qsl.base.api.entrypoint.client.ClientModInitializer

class ScryingStatuesClient : ClientModInitializer {
    override fun onInitializeClient(mod: ModContainer?) {
        EntityRendererRegistry.register(ModEntities.STATUE, ::StatueRenderer)
    }
}
